src/shared/verb-log-control.c

```diff
@@ -0,0 +1,56 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "alloc-util.h"
+#include "bus-error.h"
+#include "log.h"
+#include "strv.h"
+#include "syslog-util.h"
+#include "verb-log-control.h"
+
+int verb_log_control_common(sd_bus *bus, const char *destination, const char *verb, const char *value) {
+        assert(bus);
+        assert(endswith(verb, "log-level") || endswith(verb, "log-target"));
```

Comment:
why the asserts before the declarations?

Reply:
This function is more or less copy pasted from `log_setting_internal` which did the same so I kept it